kbfsfuse/kbfsfuse.sh

```diff
@@ -12,7 +12,7 @@ if [ -z "$KEYBASE_TEST_ROOT_CERT_PEM" ]; then
     export KEYBASE_TEST_ROOT_CERT_PEM="$(echo $KEYBASE_TEST_ROOT_CERT_PEM_B64 | base64 -d)";
 fi
 if [ -z "$KBFS_METADATA_VERSION" ]; then
-    export KBFS_METADATA_VERSION=2
+    export KBFS_METADATA_VERSION=3
```

Comment:
Oh, I was just trying it with `make test`, and it didn't work.  I guess I'll have to skip my team test when the version is 2.  But that's for another PR.  If you're not ok with changing it here by default, so it makes `make test` work, let me know.